bluemira/geometry/tools.py

```diff
@@ -210,8 +213,43 @@ def make_polygon(
     return BluemiraWire(cadapi.make_polygon(points, closed), label=label)
 
 
-@log_geometry_on_failure
 def make_bspline(
+    poles, mults, knots, periodic, degree, weights, check_rational, label: str = ""
+):
+    """
+    Builds a B-Spline by a lists of Poles, Mults, Knots
+
+    Parameters
+    ----------
+    poles: Union[list, np.ndarray]
+        list of poles.
+    mults: Union[list, np.ndarray]
+        list of integers for the multiplicity
+    knots: Union[list, np.ndarray]
+        list of knots
+    periodic: bool
+        Whether or not the spline is periodic (closed with continuous curvature?)
```

Comment:
I did mean not to have the question here. As a user, seeing a question like this in a public function would worry me, and would not inspire confidence in the code. A spline being periodic is a well-defined thing, I think we can leave it to the reader to go and look-up what that actually means, it doesn't need to be up to us to describe it here.